### PR TITLE
MISP-1055: Update no load move factors

### PR DESCRIPTION
**Pull request by jackha**
Adjusted no load move factors for testing

substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.95/gsm' generic_abs.xml.fdm_material ultimaker_abs*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.945/gsm' generic_bam.xml.fdm_material ultimaker_bam*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.935/gsm' *cffpa*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.94/gsm' *_cpe_plus*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.935/gsm' *gffpa*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.935/gsm' generic_nylon.xml.fdm_material ultimaker_nylon*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.94/gsm' generic_pla.xml.fdm_material ultimaker_pla*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}1.01/gsm' generic_pp.xml.fdm_material ultimaker_pp*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.94/gsm' generic_pva.xml.fdm_material ultimaker_pva*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}0.935/gsm' generic_tough_pla.xml.fdm_material ultimaker_tough_pla*.fdm_material
substitute.pl y 's/("no load move factor"\s*>)\d+\.*\d*/${1}1.03/gsm' generic_tpu.xml.fdm_material ultimaker_tpu*.fdm_material

generic_abs.xml.fdm_material                                                                                                                                                                     
ultimaker_abs_black.xml.fdm_material                                                                                                                                                             
ultimaker_abs_blue.xml.fdm_material                                                                                                                                                              
ultimaker_abs_green.xml.fdm_material                                                                                                                                                             
ultimaker_abs_grey.xml.fdm_material                                                                                                                                                              
ultimaker_abs_orange.xml.fdm_material                                                                                                                                                            
ultimaker_abs_pearl-gold.xml.fdm_material                                                                                                                                                        
ultimaker_abs_red.xml.fdm_material                                                                                                                                                               
ultimaker_abs_silver-metallic.xml.fdm_material                                                                                                                                                   
ultimaker_abs_white.xml.fdm_material                                                                                                                                                             
ultimaker_abs_yellow.xml.fdm_material   
generic_bam.xml.fdm_material                                                                                                                                                                     
ultimaker_bam.xml.fdm_material                                                                                                                                         
generic_cffpa.xml.fdm_material                                                                                                                              
generic_cpe_plus.xml.fdm_material                                                                                                                                    
ultimaker_cpe_plus_black.xml.fdm_material                                                                                                                     
ultimaker_cpe_plus_transparent.xml.fdm_material                                                                                                                        
ultimaker_cpe_plus_white.xml.fdm_material                                                                                                                           
generic_gffpa.xml.fdm_material                                                                                                                                      
generic_nylon.xml.fdm_material                                                                                                                                          
ultimaker_nylon_black.xml.fdm_material                                                                                                                              
ultimaker_nylon_transparent.xml.fdm_material                                                                                                                       
generic_pla.xml.fdm_material                                                                                                                                        
ultimaker_pla_black.xml.fdm_material                                                                                                                                
ultimaker_pla_blue.xml.fdm_material                                                                                                                                 
ultimaker_pla_green.xml.fdm_material                                                                                                                                  
ultimaker_pla_magenta.xml.fdm_material                                                                                                                              
ultimaker_pla_orange.xml.fdm_material                                                                                                                         
ultimaker_pla_pearl-white.xml.fdm_material                                                                                                                    
ultimaker_pla_red.xml.fdm_material                                                                                                                                     
ultimaker_pla_silver-metallic.xml.fdm_material                                                                                                                
ultimaker_pla_transparent.xml.fdm_material                                                                                                                  
ultimaker_pla_white.xml.fdm_material                                                                                                                               
ultimaker_pla_yellow.xml.fdm_material  
generic_pp.xml.fdm_material                                                                                                                                                                      
ultimaker_pp_transparent.xml.fdm_material                                                                                                                                                        
generic_pva.xml.fdm_material                                                                                                                                                                     
ultimaker_pva.xml.fdm_material
generic_pva.xml.fdm_material
ultimaker_pva.xml.fdm_material
generic_tough_pla.xml.fdm_material
ultimaker_tough_pla_black.xml.fdm_material
ultimaker_tough_pla_green.xml.fdm_material     
ultimaker_tough_pla_red.xml.fdm_material         
ultimaker_tough_pla_white.xml.fdm_material 
generic_tpu.xml.fdm_material
ultimaker_tpu_black.xml.fdm_material
ultimaker_tpu_blue.xml.fdm_material
ultimaker_tpu_red.xml.fdm_material
ultimaker_tpu_white.xml.fdm_material

### CODE REVIEW

**Comment by atrebelo** [comment]
@jackha Is this for 5.7.3-RC4?

**Comment by jackha** [comment]
> @jackha Is this for 5.7.3-RC4?

No

**Review by bajosoto — APPROVED** [review]
lgtm

File: build.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ build()
     mkdir -p "${BUILD_DIR}"
     cd "${BUILD_DIR}" || return
     echo "Building with cmake"
-    cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCPACK_PACKAGE_VERSION="${RELEASE_VERSION}" -DCPACK_PACKAGE_NAME="${PACKAGE_NAME}" -DCPACK_PACKAGE_ARCH="${ARCH}"
+    cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCPACK_PACKAGE_VERSION="${RELEASE_VERSION}" -DCPACK_PACKAGE_NAME="${PACKAGE_NAME}" -DCPACK_PACKAGE_ARCH="${ARCH}" -DEMBEDDED=ON
```
Comment (bajosoto) [inline build.sh:20]: I found a comment in deploy_to_printer.sh in Jedi-build:
`# -DEMBEDDED yields warnings for packages that don't support it, only fdm_materials`

Did you check if you get a warning enabling this flag?

**Pull request merged**